external/js/cothority/src/byzcoin/beautifier/coin.ts

```diff
@@ -0,0 +1,69 @@
+import Long from "long";
+import { Argument } from "..";
+import { Darc } from "../../darc";
+import { IBeautifyArgument } from "./utils";
+
+/**
+ * Arrange arguments for a Coin contract, ie. provide a meaningful
+ * representation of its arguments.
+ */
+export class CoinBeautifier {
+    static Spawn(args: Argument[]): IBeautifyArgument[] {
+        const res = Array<IBeautifyArgument>();
+
+        args.forEach((arg) => {
+            switch (arg.name) {
+                case "public":
```

Comment:
If you want to go the dark way, you can have:
```
case "public":
case "coinID":
case "type":
  res.push...
  break;
```

Reply:
I prefer when things are explicit (truly in this case sparing a few lines just makes the code harder to read). I don't feel like going dark today 😇